circleguard/widgets.py

```diff
@@ -318,19 +314,13 @@ def __init__(self, tooltip):
         output_combobox.setInsertPolicy(QComboBox.NoInsert)
         output_combobox.setCurrentIndex(0) # NONE by default
         self.output_combobox = output_combobox
-        self.save_folder = FolderChooser("Log Folder", get_setting("log_dir"))
```

Comment:
I agree with removing the `FolderChooser` but I would still like to have an easy way to toggle saving logs in the gui.

Reply:
for the general user or for development? I dot think we should really let people toggle logs or we might get someone reporting an issue, we ask for logs, and they don’t have any. It doesn’t do any harm to them if we save logs. 

For development I’m ok with adding it to the gui, just under the dev section.

Reply:
Well if the default is on then we don't really need the toggle, true.